### PR TITLE
docs(readme): fix typos

### PR DESCRIPTION
**Pull request by asher-gh**
closes: _please link all relevant issues_

### Check-list:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Relevant issues are marked as resolved/closed, related issues are linked in the description
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by IceTDrinker** [comment]
hello @asher-gh as you've seen we require external contributors to sign a cla to be able to contribute code to our repositories, normally the bot message has all the required information to proceed !

**Comment by IceTDrinker** [comment]
hello @asher-gh if you are not willing to sign the CLA I will close this PR as we cannot accept code without external contributors signing it.

I'll be checking back next week and if nothing changed I will close this PR.

Cheers

**Comment by asher-gh** [comment]
already signed the CLA. Is there a way to recheck that?

**Comment by IceTDrinker** [comment]
let me check internally @asher-gh did you sign it for a bounty by chance ?

**Comment by IceTDrinker** [comment]
@cla-bot check

**Comment by IceTDrinker** [comment]
all good !

**Comment by IceTDrinker** [comment]
@slab-ci cpu_fast_test

File: README.md [hunk 1]
```diff
@@ -92,10 +93,10 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // On the server side:
     set_server_key(server_keys);
 
-    // Clear equivalent computations: 1344 * 8 = 10752
+    // Clear equivalent computations: 1344 * 5 = 6720
```
Comment (IceTDrinker) [inline README.md:96]: this is to match the change below ?

File: README.md [hunk 2]
```diff
@@ -18,53 +18,54 @@
...
 tfhe = { version = "*", features = ["boolean", "shortint", "integer", "aarch64-unix"] }
 ```
-Note: users with ARM devices must use `TFHE-rs` by compiling using the `nightly` toolchain.
 
+Note: users with ARM devices must use `TFHE-rs` by compiling using the `nightly` toolchain.
 
-+ For x86_64-based machines with the [`rdseed instruction`](https://en.wikipedia.org/wiki/RDRAND) 
-running Windows:
+- For x86_64-based machines with the [`rdseed instruction`](https://en.wikipedia.org/wiki/RDRAND)
+  running Windows:
```
Comment (IceTDrinker) [inline README.md:56]: double space -> single space

File: README.md [hunk 3]
```diff
@@ -116,20 +117,20 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
...
 
-
 ## Contributing
 
 There are two ways to contribute to TFHE-rs:
 
 - you can open issues to report bugs or typos, or to suggest new ideas
 - you can ask to become an official contributor by emailing [hello@zama.ai](mailto:hello@zama.ai).
-(becoming an approved contributor involves signing our Contributor License Agreement (CLA))
+  (becoming an approved contributor involves signing our Contributor License Agreement (CLA))
```
Comment (IceTDrinker) [inline README.md:133]: double space -> single space

**Review by IceTDrinker — COMMENTED** [review]
Thanks for checking our README file !

A few very minor comments

**Comment by IceTDrinker** [comment]
@asher-gh we don’t use merge strategies for our PRs, we prefer rebasing on main, so you can just add your commit on top of the current main branch

**Review by IceTDrinker — APPROVED** [review]
Thanks !

**Comment by IceTDrinker** [comment]
thanks for the contribution @asher-gh !

**Pull request merged**